### ISSUE TITLE
Middleware

### ISSUE THREAD

**Issue by Sephoric1994** [aelassas/bookcars#44]
Hi! So while I was editing the database, I realised that when I change the car values in the database, the front end cars do not show anymore. However, the changes are reflected on the backend. So when I update the car changes from the backend, the front end will reflect after that. How do i remove this middleware that's passing message from backend to frontend? and enable the frontend to get the data from the database directly?

**Comment by aelassas** [comment]
>  I realised that when I change the car values in the database, the front end cars do not show anymore.

Which values did you change?

> How do i remove this middleware that's passing message from backend to frontend? and enable the frontend to get the data from the database directly?

There is no middleware between the backend and the frontend. The frontend retrieves data directly from the database through the api. The backend stores the data directly in the database through the api.

**Comment by Sephoric1994** [comment]
> #Which values did you change?

For instance, when I change the vehicles names from "BMW X1" to "Toyota Vios", the vehicle disappears from the frontend despite the fact that the pickup locations of the value remains the same. But when I go to the backend, and click on the vehicle that I changed to "Toyota Vios", remove one of the location and click update, I can search for the vehicle in the front end based on the 2 locationvalues left in the DB. I exported the car table out, edited the whole list of 40 vehicles to 39 vehicle names based on what my company has, re-imported the table in and set the locations from "String" to "ObjectId" following the table structure, the front end shows No Car even when i searched on the same location.

Actually I'm looking to totally remove the locations from the retrieval point. Such that I'll only have 2 modes, Self-Collection/Delivery, and users are able to see the whole list of vehicles instead. It'll only be for creation of a new booking.
